### PR TITLE
Use namespace specified in operator config file correctly

### PR DESCRIPTION
**Pull request by irajdeep**
### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +21,6 @@ DOCKERFILE ?= operator
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,crdVersions=v1"
 RELEASE_NAME_HELM ?= mongodb-kubernetes-operator
-TEST_NAMESPACE ?= mongodb
```
Comment (lsierant) [inline Makefile:22]: or here: it could be just TEST_NAMESPACE = $(NAMESPACE)

File: Makefile [hunk 2]
```diff
@@ -10,6 +10,8 @@ UPGRADE_HOOK_IMG := $(shell jq -r .version_upgrade_hook_image < $(MONGODB_COMMUN
 READINESS_PROBE_IMG := $(shell jq -r .readiness_probe_image < $(MONGODB_COMMUNITY_CONFIG))
 REGISTRY := $(shell jq -r .repo_url < $(MONGODB_COMMUNITY_CONFIG))
 AGENT_IMAGE_NAME := $(shell jq -r .agent_image_ubuntu < $(MONGODB_COMMUNITY_CONFIG))
+TEST_NAMESPACE := $(shell jq -r .namespace < $(MONGODB_COMMUNITY_CONFIG))
```
Comment (lsierant) [inline Makefile:13]: it could be just TEST_NAMESPACE = $(NAMESPACE)

**Review by slaskawi — APPROVED** [review]
LGTM

**Review by lsierant — COMMENTED** [review]
Generally I have this in handled in #1066

File: Makefile (hunk 2 above)
Comment (irajdeep) [reply Makefile:13]: we could just use NAMESPACE and get rid of TEST_NAMESPACE.

**Comment by irajdeep** [comment]
closing in favor of: https://github.com/mongodb/mongodb-kubernetes-operator/pull/1066

**Pull request closed without merging**